content/SCALE/GettingStarted/Configure/FirstTimeLogin.md

```diff
@@ -153,28 +153,28 @@ If the web interface displays but seems unresponsive or incomplete:
 If the UI becomes unresponsive after an upgrade or other system operation, clear the site data and refresh the browser (<kbd>Shift</kbd>+<kbd>F5</kbd>).
 {{< /expand >}}
 
-## Introducing the Main SCALE Dashboard
+## Introducing the SCALE Dashboard
 
-{{< expand "Dashboard Video Tutorial" >}}
-
-{{< embed-video name="scaledashboardtour" >}}
-
-{{< /expand >}}
 After logging in for the first time, the main system **Dashboard** screen displays.
-The **Dashboard** shows different system information cards (widgets) with basic information about the installed version, systems component usage, network traffic, and configured pools or storage usage. 
+The **Dashboard** shows different system information cards (widgets) with basic information about the installed version, systems component usage, network traffic, and configured pools or storage usage.
+The dashboard includes new categories of widgets that include a text-only **Custom** widget, and an **Apps** widget you can configure to monitor your installed applications.
 
 {{< enterprise >}}
-SCALE Enterprise users with an iXsystems-provided TrueNAS server also see an image of the system in the **System Information** widget. Click on the system image to open the **System > [View Enclosure]({{< relref "EnclosureScreensSCALE.md" >}})** screen.
+SCALE Enterprise users with an iXsystems-provided TrueNAS server also see an image of the system in the **System Information** widget.
+Click on the system image to open the **System Settings > [View Enclosure]({{< relref "EnclosureScreensSCALE.md" >}})** screen.
+
 
-{{< trueimage src="/images/SCALE/Dashboard/DashboardMainScreenSCALE.png" alt="TrueNAS SCALE Dashboard" id="TrueNAS SCALE Dashboard" >}}
+{{< trueimage src="/images/SCALE/Dashboard/DashboardMainScreen.png" alt="TrueNAS SCALE Dashboard" id="TrueNAS SCALE Dashboard" >}}
```

Comment:
I want to use the image of the dashboard here, not just the system information widget because this section is about the dashboard and not just about the system information widget. I've move the image up above the enterprise shortcode section, and I removed the enterprise short codes.